src/components/Autocomplete/Autocomplete.tsx

```diff
@@ -0,0 +1,99 @@
+import { FC, useState } from 'react';
+import cn from 'classnames';
+import { debounce } from '../../helper';
+import { Person } from '../../types/Person';
+import './Autocomplete.scss';
+
+type Props = {
+  options: Person[],
+  getValue: (value: Person) => void;
+};
+
+const getCurrentOptions = (options: Person[], query: string): Person[] | [] => {
+  if (query) {
+    return options.filter(({ name }) => {
+      return name.toLowerCase().includes(query.toLowerCase());
+    });
+  }
+
+  return [...options];
+};
+
+export const Autocomplete: FC<Props> = ({ options, getValue }) => {
+  const [showMenu, setShowMenu] = useState(false);
+  const [currentValue, setCurrentValue] = useState('');
+  const [searchQuery, setSearchQuery] = useState('');
+  const currentOptions = getCurrentOptions(options, searchQuery);
+
+  const debouncedSearchQuery = debounce<string>((arg) => {
+    setSearchQuery(arg);
+  }, 1000);
+
+  const handleInputOnChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const { value } = e.target;
+
+    setCurrentValue(value);
+    debouncedSearchQuery(value);
+  };
+
+  const handleInputOnBlur = () => {
+    setTimeout(() => setShowMenu(false), 200);
+  };
+
+  const handleSelectedValue = (value: string) => {
+    setCurrentValue(value);
+
+    getValue(currentOptions.find(({ name }) => {
+      return name === value;
+    }) as Person);
+
+    setShowMenu(false);
+  };
+
+  return (
+    <div className="dropdown is-active">
+      <div className="dropdown-trigger">
+        <input
+          type="search"
+          placeholder="Enter a part of the name"
```

Comment:
The placeholder text does not fit completely into the input
<img width="248" alt="image" src="https://github.com/mate-academy/react_autocomplete/assets/54860641/8d24ed2f-d13d-4666-85cc-04f88c62f042">